guacamole-docker/bin/start.sh

```diff
@@ -1064,6 +1069,11 @@ fi
 set_property "guacd-hostname" "$GUACD_HOSTNAME"
 set_property "guacd-port"     "$GUACD_PORT"
 
+# A comma-separated list of the identifiers of authentication providers that
+# should be allowed to fail internally without aborting the authentication process
+set_property "skip-if-unavailable"     "$SKIP_IF_UNAVAILABLE"
```

Comment:
Shouldn't this be `set_optional_property`? Using `set_property` will put an empty string in there or possibly other unexpected results if it isn't present.

Reply:
makes sense!